release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+
+[id="ocp-4-7-18"]
+=== RHBA-2021:2502 - {product-title} 4.7.18 bug fix update
+
+Issued: 2021-06-28
+
+{product-title} release 4.7.18 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2502[RHBA-2021:2502] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:R2503[RHBA-2021:2503] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6139672[{product-title} 4.7.18 container image list]
+
+[id="ocp-4-7-18-features"]
+==== Features
+
+[id="ocp-4-7-18-builds-telemetry-metric"]
+===== New Telemetry metric for number of builds by strategy
+In this update, Telemetry includes a new `openshift:build_by_strategy:sum` gauge metric, which sends the number of builds by strategy type to the Telemeter Client. This metric gives site reliability engineers (SREs) and product managers visibility into the kinds of builds that run on {product-title} clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1969963[*BZ#196963*])
```

Comment:
BZ # doesn't match here. Looks like "196963" should be "1969963".

Reply:
Agreed! Changed!